apps/console/src/features/users/components/guests/models/invite.ts

```diff
@@ -39,6 +39,18 @@ export interface UserInviteInterface {
     username?: string;
 }
 
+/**
+ * Interface to represent invite for parent org user.
+ */
+export interface ParentOrgUserInviteInterface {
+    id?: string;
+    roles?: string[];
```

Comment:
Roles is a mandatory attr. when inviting a user from parent org AFAIR. If yes, let's not mark this as optional.

Reply:
roles is not a mandatory attribute